internal/events/sql.go

```diff
@@ -59,6 +73,9 @@ const (
 		DELETE FROM ` + kvTableName + `
 		WHERE key = $1;
 	`
+
+	// V0 to V1 migration
+	dropReceivedColumn = `ALTER TABLE ` + schemaName + `.events DROP COLUMN received;`
```

Comment:
`received` is still in the `CREATE TABLE` statement.  Wait, does our store initialization on a fresh machine run through all the upgrades rather than just starting at the current version?

Reply:
This is what confusing to me in your PR,. We actually go from version 0 to current. I had it earlier to just start at latest, but I had a strong pushback to go from 0 to latest. So we shouldn't update the tables like u did in yours

Reply:
This is unrelated to the ability to support existing schema that just lack the version table, which we can live without.

Reply:
Oh, I guess we have to decide about if a new node should run through all the upgrades in empty tables.

But at far as redefining v1 upgrade function, that's ok.  It has only been a couple days since it was defined, and I've not deployed it.  If we had CD or nightly builds then that would be a no no, but it's fine for now.

Reply:
On second thought, about the v1 redefine, a consideration is if we'd like the meta data update on the release branch but not this vote store refactor.  Just for instance.  I don't know the answer to that